src/Microsoft.AspNetCore.Sockets.Client.Http/HttpConnection.cs

```diff
@@ -42,6 +42,10 @@ public HttpConnection(Uri url)
             : this(url, TransportType.WebSockets)
         { }
 
+        public HttpConnection(Uri url, HttpMessageHandler httpMessageHandler)
+            : this(url, TransportType.WebSockets, loggerFactory: null, httpMessageHandler: httpMessageHandler)
```

Comment:
This should be TransportType.All. I think you should fix it everywhere and we will add the test that is failing in my PR that did that separately.